src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -140,12 +141,17 @@ trait AttributeComponent {
           val attributeRecsWithRefForName: Set[(AttributeRecord, Option[EntityRecord])] = attributeRecsWithRefForNameWithDupes.map { case ((wsId, attributeRec), entityRec) => (attributeRec, entityRec) }.toSet
           if (attributeRecsWithRefForName.forall(_._1.listIndex.isDefined)) {
             unmarshalList(attributeRecsWithRefForName)
-          } else if (attributeRecsWithRefForName.size > 1) {
-            throw new RawlsException(s"more than one value exists for attribute but list index is not defined for all, records: $attributeRecsWithRefForName")
-          } else if (attributeRecsWithRefForName.head._2.isDefined) {
-            unmarshalReference(attributeRecsWithRefForName.head._2.get)
           } else {
-            unmarshalValue(attributeRecsWithRefForName.head._1)
+            //HACK: Work around corrupted data due to GAWB-443. If there are duplicate records for this attribute,
+            //log an error (not an exception) and then just take the first one.
+            if (attributeRecsWithRefForName.size > 1) {
+              logger.error(s"more than one value exists for attribute but list index is not defined for all records: $attributeRecsWithRefForName")
```

Comment:
If we're gonna push this out as a hotfix, could you add additional logging to help us diagnose? I'd like to see:
* `attributeRecsWithRefForName.size` logged
* `attributeRecsWithRefForName.collect(_._1.listIndex.isEmpty).size` logged (writing code in github, may not compile)

I realize we can pull this info out of the big $attributeRecsWithRefForName string, but this would be easier